src/components/icons/EyeShowPassword.tsx

```diff
@@ -0,0 +1,9 @@
+const EyeShowPassword = () => {
+    return (
+        <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same comment as before, would be better to just include awesomefont (or any other existing library) so we can maintain same look&feel in the future? :)